### PR TITLE
chore: tidy script files

### PR DESCRIPTION
**Pull request by parkjh**

### CODE REVIEW

**Comment by parkjh** [comment]
this seems minor but this is a prelude to another upcoming pr...

**Comment by ValGeorgiev** [comment]
What will be the upcoming PR, what is the goal here?

**Comment by parkjh** [comment]
@ValGeorgiev extract the dependancies of the cli and fetch font into a separate packages, and ideally there shouldn't be any npm install for fetch font but instead use pure node api
end result = smaller (dev)dependancy of the overall monorepo

File: lib/scripts/publish.sh [hunk 1]
```diff
@@ -67,4 +67,4 @@ lerna publish --conventional-commits --yes --concurrency=1 --exact -m "$MESSAGE"
 echo "Pushing to master"
 git push origin master --tags --quiet > /dev/null 2>&1
 
-./lib/publish_storybook.sh
+./lib/scripts/publish_storybook.sh
```
Comment (parkjh) [inline lib/scripts/publish.sh]: @milesillsley @MarkConway94 is the release cli package looking for this script in a hardcoded path directory?

Comment (tuncaulubilge) [reply lib/scripts/publish.sh]: All the CI scripts should be using yarn commands, not hardcoded paths, but please do check bitrise @parkjh

Comment (parkjh) [reply lib/scripts/publish.sh]: @tuncaulubilge I've updated the all the yarn commands but I could not find any reference to this script, but if it's in bitrise I will create the yarn command to replace it

Comment (tuncaulubilge) [reply lib/scripts/publish.sh]: `"storybook:publish": "./lib/publish_storybook.sh",` in the root package.json

Comment (parkjh) [reply lib/scripts/publish.sh]: https://github.com/newsuk/times-components/blob/0da0a1e74d90fdb6b9c8913f4e304aeb0dc41985/package.json

Comment (parkjh) [reply lib/scripts/publish.sh]: @tuncaulubilge confirmed with @milesillsley that the release cli is not used in bitrise for TC, so I will just delete this I think

Comment (milesillsley) [reply lib/scripts/publish.sh]: Bitrise as no direct reference to this script, and pretty sure we don't have release cli on TC

Comment (tuncaulubilge) [reply lib/scripts/publish.sh]: Does bitrise have any references to any scripts? It should be using commands but I want to make sure we don't miss anything

Comment (parkjh) [reply lib/scripts/publish.sh]: @tuncaulubilge https://app.bitrise.io/app/b82f579b5213b496#/workflow no `sh` references as I can see...

Comment (parkjh) [reply lib/scripts/publish.sh]: they just copied and pasted the script file content into the step itself...

Comment (tuncaulubilge) [reply lib/scripts/publish.sh]: 👍

**Pull request closed without merging**